ui/src/app/features/application-header/application-header.facade.ts

```diff
@@ -35,21 +35,24 @@ import {
 export class ApplicationHeaderFacade implements IFacade {
   selectedId$: Observable<string | null>;
   selectedId: string | null;
-  loading$: Observable<boolean>;
+  isLoading$: Observable<boolean>;
   userRole$: Observable<string | null>;
+  currentUserRole$: Observable<string | null>;
```

Comment:
please rename to globalRole$ or organisationRole$

Reply:
Fixed